app/src/main/java/me/saket/dank/ui/accountmanager/AccountManager.java

```diff
@@ -0,0 +1,68 @@
+package me.saket.dank.ui.accountmanager;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+
+import com.google.auto.value.AutoValue;
+import io.reactivex.functions.Function;
+
+import me.saket.dank.ui.accountmanager.AccountManagerScreenUiModel;
+import me.saket.dank.ui.accountmanager.AutoValue_AccountManager;
+import me.saket.dank.utils.Cursors;
+
+@AutoValue
+public abstract class AccountManager implements AccountManagerScreenUiModel {
+
+  static final String TABLE_NAME = "Account";
+  static final String COLUMN_USERNAME = "username";
```

Comment:
This column is not clear to me.

1. The type in create table query is set to text
2. Mapper maps it to int
3. Mapper reads it as user
4. Create method uses it as rank
5. toValues method also uses it as rank
6. WHERE_USERNAME doesn't use it but uses label column

You see the inconsistencies? I guess what you want here are columns named rank and username instead. You should use the same naming everywhere.